tools/wego-app-dev.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wego-app
+  namespace: wego-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: wego-app
+    spec:
+      serviceAccountName: wego-app-service-account
+      containers:
+        - name: wego-app
+          image: ghcr.io/weaveworks/wego-app:latest
```

Comment:
This should be a local deployment. This still will pull in the latest and require rebuilding of the image. It won't use what the local rebuild did. It should be put into a local kind repository and replace `ghcr.io` with `localhost:5000`.

Reply:
Your can edit the tilt file to do a prebuild exercise where it does a `sed` replace in place in the file with the correct domain.

Reply:
Maybe that's out of scope of this ticket for now. It's for you to decide. :)

Reply:
> This still will pull in the latest and require rebuilding of the image

The behavior I am seeing is that changes to the code result in them ending up in the cluster `wego-app`. Is that not happening for you @Skarlso ?

Reply:
Ok fixed to use the local registry

Reply:
Awesome!